uiowa/uiowa
# ITS Services View Block

Resolves https://github.com/uiowa/uiowa/issues/6826

<img width="1364" alt="Screenshot 2023-09-11 at 2 48 19 PM" src="https://github.com/uiowa/uiowa/assets/4663676/49d2b561-9dbe-4263-8478-5f38381eb1e0">

# To Test
```
ddev blt frontend && rm config/sites/its.uiowa.edu/schemadotorg.schemadotorg_mapping.node.service.yml && ddev blt ds --site=its.uiowa.edu && ddev drush @its.local config:import --source ../config/sites/its.uiowa.edu --partial -y && ddev drush @its.local cim -y && git restore config/ && ddev drush @its.local cim -y && ddev drush @its.local cr && ddev drush @its.local uli /admin/structure/taxonomy/manage/service_category/add
```
- Create some audience/category terms
- Create some services. Create a few services that have a leading integer.
- Add a search block to a layout with  `/all-services-and-software` and `name` as the config. Try searching on a partial title.
- See that styling is not too much of a departure from https://sandbox.prod.drupal.uiowa.edu/services and note https://github.com/uiowa/uiowa/issues/6826#issuecomment-1714228825
